src/plugins/loan-channel-query-map.js

```diff
@@ -265,7 +315,47 @@ export default {
 					url: 'world-refugee-day',
 					queryParams: 'status=fundRaising&riskRating=0,5&theme=Refugees/Displaced',
 					algoliaParams: 'attributes=Refugees%2FDisplaced',
-					flssLoanSearch: { themeId: [28] },
+					flssLoanSearch: {
+						themeId: [28]
+					},
+				},
+				{
+					id: 72,
+					url: 'loans-to-artisans',
+					queryParams: 'status=fundRaising&sector=9',
+					flssLoanSearch: {
+						sectorId: [9]
+					},
+				},
+				{
+					id: 74,
+					url: 'loans-for-food-producers',
+					queryParams: 'status=fundRaising&sector=12',
+					flssLoanSearch: {
+						sectorId: [12]
+					},
+				},
+				{
+					id: 75,
+					url: 'loans-to-underbanked-areas',
+					queryParams: 'theme=Underfunded Areas',
+					flssLoanSearch: {
+						themeId: [13]
+					},
+				},
+				{
+					id: 77,
+					url: 'loans-for-retail-businesses',
+					queryParams: 'status=fundRaising&sector=7',
+					flssLoanSearch: {
+						sectorId: [7]
+					},
+				},
+				{
+					id: 79,
+					url: 'group-loans',
+					queryParams: 'status=fundRaising&isGroup=1&distributionModel=both',
+					// flssLoanSearch: {},
```

Comment:
Remove?

Reply:
I just left as a placeholder and figured we'd add it once we include that param.